core/storage/opportunity/gae_models.py

```diff
@@ -256,3 +256,175 @@ def get_skill_opportunities(
             (cursor.urlsafe().decode('utf-8') if cursor else None),
             more_results
         )
+
+
+class PinnedOpportunityModel(base_models.BaseModel):
```

Comment:
This is personal to a user, right, and is more of a user preference? If so, let's put it in user_models.py.

Reply:
Done, shifted this model to userModels